pkg/netutils/netutils.go

```diff
@@ -211,16 +211,11 @@ func IsValidIP(ip string) bool {
 	if strings.TrimSpace(ip) == "" {
 		return false
 	}
-
-	// str is a regex which matches a digital
-	// greater than or equal to 0 and less than or equal to 255
-	str := "(?:25[0-5]|2[0-4]\\d|[01]?\\d?\\d)"
-	result, err := regexp.MatchString("^(?:"+str+"\\.){3}"+str+"$", ip)
-	if err != nil {
+	testInput := net.ParseIP(ip)
+	if testInput.To4() == nil && testInput.To16() == nil {
```

Comment:
> If s is not a valid textual representation of an IP address, ParseIP returns nil.

So it should test the result of `ParseIP` whether is nil here.

Reply:
thanks ,updated